Repo-Integration/setup.sh

```diff
@@ -85,7 +152,8 @@ then
     echo "${cyn}export ws_key='replace-with-your-activation-key-inside-single-quotes'${end}"
     exit
 else
-    scm
+    scm;
+    cert_add;
```

Comment:
Cert_add should only fire if $2 is supplied

Reply:
This is taken care of by first if/fi statement in the cert_add function